hibernate-reactive-core/src/test/java/org/hibernate/reactive/WrongCredentialsTest.java

```diff
@@ -8,36 +8,35 @@
 import javax.persistence.Id;
 
 import org.hibernate.cfg.Configuration;
+import org.hibernate.internal.CoreLogging;
+import org.hibernate.internal.CoreMessageLogger;
 import org.hibernate.reactive.provider.Settings;
-import org.hibernate.reactive.testing.DatabaseSelectionRule;
 
-import org.junit.Rule;
 import org.junit.Test;
 
 import io.vertx.ext.unit.TestContext;
 
-import static org.hibernate.reactive.containers.DatabaseConfiguration.DBType.POSTGRESQL;
-
 /**
  * Check that the right exception is thrown when there is an error with the credentials.
  * <p>
  *     Similar to {@link org.hibernate.reactive.configuration.ReactiveConnectionPoolTest} but at the session level.
- *     Note that the wrong credentials are also used for the schema generation but exceptions are ignored during
- *     schema generation. You might just see some warnings in the log.
+ *     Note that the wrong configuration is also used for the schema generation but ORM catch the exception and
+ *     continue anyway it in case of errors. You might just see sine warnings in the log.
```

Comment:
Wait, did you change the javadoc or just copied and pasted the one in the old test?

Reply:
copy paste... reverted text